misc/add-lxc-iptag.sh

```diff
@@ -133,8 +133,9 @@ if [[ ! -f /opt/lxc-iptag/iptag.conf ]]; then
 # List of allowed CIDRs
 CIDR_LIST=(
   192.168.0.0/16
-  100.64.0.0/10
+  172.16.0.0/12
   10.0.0.0/8
+  100.64.0.0/10
```

Comment:
This is not a RFD1918 Address Range, but i know it has been in there befor your change. But i dont now why tbh.
```suggestion
```